components/Heading.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import FormatListBulletedIcon from "@material-ui/icons/FormatListBulleted";
+
+const HeadingContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+`;
+
+const HeadingBox = styled.div`
+  padding-top: 40px;
+  padding-bottom: 30px;
+  color: #fff;
+  text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.4);
```

Comment:
Its a good practice to use one type of color (either hash or rgba), and to extract color from theme file will be a good practice.